### PR TITLE
Highcharts - Add maxFontSize and minFontSize to series that have it.

### PR DESCRIPTION
**Pull request by justinmwarner**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://api.highcharts.com/highcharts/series.line.label.maxFontSize
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by justinmwarner** [comment]
@RyanCavanaugh travis issue here. Can you advise?

File: types/highcharts/index.d.ts [hunk 1]
```diff
@@ -5420,7 +5484,16 @@ declare namespace Highcharts {
         width?: number | string;
     }
 
-    interface ScatterChart extends SeriesChart { }
+    interface ScatterChart extends SeriesChart {
+        /**
+         * For area-like series, allow the font size to vary so that small areas get a smaller font size. The default applies this effect to area-like series but not line-like series.
+         */
+        maxFontSize?: number;
```
Comment (DanielRosenwasser) [inline types/highcharts/index.d.ts]: Are all of these just part of `SeriesChart`s? If not, why not extract them to their own interface and avoid the duplication?

File: types/highcharts/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Highcharts 5.0.13
+// Type definitions for Highcharts 5.0.32
```
Comment (DanielRosenwasser) [inline types/highcharts/index.d.ts]: Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
Sorry about the Travis issue; things should be good now, and thanks for being patient as we go through these PRs.

**Pull request closed without merging**